module/organizationExtension/helper.js

```
@@ -0,0 +1,166 @@
+				)
+
+				// Get all allowed values for org extension visibility
+				let orgExtenVisibilityValues = Object.values(CONSTANTS.common.ORG_EXTENSION_VISIBILITY)
+
+				// Validate each provided visibility policy
+				Object.entries(filteredBodyData).map(([key, value]) => {
+					if (key == 'projectResourceVisibilityPolicy' || key == 'externalProjectResourceVisibilityPolicy') {
+						value = value.toUpperCase()
+						// If provided value is not valid, reset to default
+						if (!orgExtenVisibilityValues.includes(value))
+							filteredBodyData[key] = CONSTANTS.common.DEFAULT_ORG_EXTENSION_POLICIES[key]
```

ESLint

[error] 'CONSTANTS' is not defined.

(no-undef)